cmd/exec.go

```diff
@@ -78,9 +78,12 @@ func Exec() *cobra.Command {
 
 func executeExec(ctx context.Context, dev *model.Dev, args []string) error {
 
+	wrapped := []string{"sh", "-c"}
```

Comment:
@pchico83 do you see any potential side effects on this approach?

Reply:
not having `sh`, but we already have that problem everywhere

Reply:
true. we need to document that for now.